ui/apps/platform/cypress/integration/networkGraph/networkGraph.test.js

```diff
@@ -55,10 +54,7 @@ describe('Network Graph Search', () => {
         });
     });
 
-    it('should filter to show only a specific deployment and deployments connected to it', function () {
-        if (hasFeatureFlag('ROX_POSTGRES_DATASTORE')) {
-            this.skip();
-        }
+    it('should filter to show only a specific deployment and deployments connected to it', () => {
```

Comment:
Any reason for the change from `function ()` to `() =>` ?

Reply:
Linter :)
Apart from that I changed it from `() =>` to `function ()` to be able to access `this` and `skip()` the test.